bindings/python/tests/conftest.py

```diff
@@ -83,28 +83,74 @@ def fa_one_divisible_by_eight():
 @pytest.fixture(scope="function")
 def fa_odd_ones():
     lhs = mata.Nfa(2)
-    lhs.add_initial_state(0)
+    lhs.make_initial_state(0)
     lhs.add_trans_raw(0, 0, 0)
     lhs.add_trans_raw(0, 1, 1)
     lhs.add_trans_raw(1, 1, 0)
     lhs.add_trans_raw(1, 0, 1)
-    lhs.add_final_state(1)
+    lhs.make_final_state(1)
 
 
 @pytest.fixture(scope="function")
 def fa_even_ones():
     lhs = mata.Nfa(2)
-    lhs.add_initial_state(0)
+    lhs.make_initial_state(0)
     lhs.add_trans_raw(0, 0, 0)
     lhs.add_trans_raw(0, 1, 1)
     lhs.add_trans_raw(1, 1, 0)
     lhs.add_trans_raw(1, 0, 1)
-    lhs.add_final_state(0)
+    lhs.make_final_state(0)
 
 
 @pytest.fixture(scope="function")
 def binary_alphabet():
     alph = mata.OnTheFlyAlphabet()
     alph.translate_symbol("0")
     alph.translate_symbol("1")
-    yield alph
\ No newline at end of file
+    yield alph
+
+
+def fill_with_automaton_a(nfa):
```

Comment:
I would consider making this a fixture (see line 105), with `function` scope, i.e. for each test it will create new automaton.

Reply:
Done. Good suggestion.